modules/admin_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc

```diff
@@ -36,52 +56,42 @@ You can see an example of calling the commands listed below, configured to requi
 NOTE: The decryption of the files by the ownCloud administrator requires the current passwords of all users!
 This only works when users have enabled password recovery and if an admin recovery password is available.
 
-== Remove the Encryption Records from the ownCloud Database
+=== Remove the Encryption Records from the ownCloud Database
 
 Once your ownCloud files are unencrypted, and encryption has been disabled, you need to remove the encryption records from the database.
 There is, currently, no `occ` command to handle this, so it has to be done manually.
 Specifically, you need to remove all records from the `oc_appconfig` table where the `appid` column is set to `encryption`.
 
-In the examples below you can see how to do this using the SQLite database.
-If you are not using SQLite, please use the commands specific to your database vendor.
-
-NOTE: The example code assumes that the path to the SQLite database is
-`<YOUR/OWNCLOUD/ROOT/DIRECTORY>data/owncloud.database`.
-
-[source,console]
-----
-sudo sqlite3 data/owncloud.database
-sqlite> delete from oc_appconfig where appid='encryption';
-sqlite> select * from oc_appconfig where appid='encryption';
-----
-
-Same for mysql:
+In the examples below you can see how to do this using MySQL.
+If you are not using MySQL, please use the commands specific to your database vendor.
 
+[source,sql]
 ----
 SELECT * FROM `oc_appconfig` WHERE `appid` LIKE 'encryption'
 ----
 
-== Remove the `files_encryption` Directory
+=== Remove the `files_encryption` Directory
 
 With the database updated, next, the `files_encryption` directory needs to be removed.
 Below is an example of how to do so, to save you time.
 
 [source,console]
 ----
+cd <your owncloud root directory>
 find ./data* -name files_encryption -exec rm -rvf {} \;
 ----
 
-== Encrypt the Filesystem Using Master Key-based Encryption
+=== Encrypt the Filesystem Using Master Key-based Encryption
 
 Now, your ownCloud files can be encrypted using Master Key-based encryption.
 This requires the following steps:
 
 . The encryption app needs to be enabled
 . Encryption needs to be enabled
-. The encryption type needs to be set to master key
+. The encryption type needs to be set to btn:[Master Key]
```

Comment:
The `btn` formatting and the wording of the sentence aren't in sync. Please update the sentence's wording to tell the user that they have to click a button.